aries_vcx/src/common/proofs/prover/prover_internal.rs

```diff
@@ -15,6 +15,7 @@ pub struct CredInfoProver {
     pub revocation_interval: Option<NonRevokedInterval>,
     pub tails_file: Option<String>,
     pub timestamp: Option<u64>,
+    pub revealed: bool
 }
 
 pub async fn build_schemas_json_prover(
```

Comment:
bonus points to add test checking that `build_schemas_json_prover` produces `revealed: true` by default and `revealed: false` if overriden